tests/runtime/other

```diff
@@ -53,7 +53,7 @@ RUN bpftrace -v -e 'BEGIN { if (str($1) != str($2)) { printf("I got %s\n", str($
 EXPECT not equal
 TIMEOUT 5
 
-NAME string compare map lookup
+NAME string_compare_map_lookup
```

Comment:
curious why this is necessary

Reply:
Whitespace was messing with the test ignore: https://github.com/iovisor/bpftrace/pull/1110/files#diff-117779d01dcaa4bec09a6b3a71b8c8bdR64

Might just be me and quoting again, can fix this if preferred.

Reply:
ahh i didn't see you using it above, makes sense.

Reply:
yep, quoting -_- fixed